requirements/base.txt

```diff
@@ -68,3 +68,4 @@ raven==6.9.0
 collectfast==0.6.2
 django-health-check==3.7.0
 elastic-apm==3.0.0
+https://github.com/pinkiebell/wagtail/archive/master.tar.gz
```

Comment:
Why not `wagtail` or `git+https://github.com/wagtail/wagtail.git#egg=wagtail` ?

Reply:
We need the 2.3 release(for Django 2.1 support) which is currently under development and should be released in October as a LTS release.
Wagtail needs npm for building, so I pushed a prebuilt version to my repo.